Dockerfile.template

```diff
@@ -21,13 +21,13 @@ RUN	rm -rf /var/lib/mysql && \
 		| xargs -0 grep -lZE '^(bind-address|log|user\s)' \
 		| xargs -rt -0 sed -Ei 's/^(bind-address|log|user\s)/#&/' || true
 
-RUN mkdir /docker-entrypoint-initdb.d
-
 COPY docker-entrypoint.sh /usr/local/bin/
-RUN ln -s usr/local/bin/docker-entrypoint.sh / # backwards compat
 
-RUN chmod +x /usr/local/bin/docker-entrypoint.sh && \
-	chown -R mysql:mysql /usr/local/bin/docker-entrypoint.sh
+# symlink for backwards compatiblity
```

Comment:
These don't need to be separate `RUN` statements, which creates additional layers.

Reply:
This is interesting, when I tried this locally before I published the image this would cause problems. The job passed here though 🤷‍♂️  LGTM